src/app/qgisapp.cpp

```diff
@@ -13125,6 +13112,96 @@ void QgisApp::unregisterDevToolFactory( QgsDevToolWidgetFactory *factory )
   mDevToolFactories.removeAll( factory );
 }
 
+
+void QgisApp::showApiDocumentation( const QString &api, bool embedded, const QString &object, const QString &module )
```

Comment:
Could you move the guts of this to QgsDocumentationPanelWidget ? I'd like to reduce the amount of code in qgisapp.cpp, which is already ridiculously large